Index: black2/docs/conf.py
===================================================================
--- black2.orig/docs/conf.py
+++ black2/docs/conf.py
@@ -117,7 +117,7 @@ pygments_style = "sphinx"
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "furo"
+#html_theme = "furo"
 html_logo = "_static/logo2-readme.png"
 
 # Add any paths that contain custom static files (such as style sheets) here,
