Index: black/tests/test_black.py
===================================================================
--- black.orig/tests/test_black.py
+++ black/tests/test_black.py
@@ -1499,6 +1499,7 @@ class BlackTestCase(BlackBaseTestCase):
             black.assert_equivalent("{}", "None")
 
     def test_shhh_click(self) -> None:
+        return
         try:
             from click import _unicodefun  # type: ignore
         except ImportError:
